ajax/libs/jquery.touch/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "jquery.touch",
+  "filename": "jquery.touch.min.js",
+  "version": "1.1.0",
+  "description": "Adds touch gesture events to jQuery",
+  "keywords": [
+    "jquery",
+    "jquery touch"
```

Comment:
Please just use `touch`. We prefer the condition that there is no space between a single keyword expect it has to be more than two words to express specific meaning e.g "Computer Science". Thanks.